.travis.yml

```diff
@@ -1,12 +1,41 @@
 sudo: required
+dist: trusty
+language: python
+matrix:
+    include:
+        - python: 2.7
```

Comment:
I don't think that tests are actually running on Python 2.7 — for the most recent job, the `pytest` runner is actually [executing on Python 3.6](https://travis-ci.org/ubclaunchpad/minutes/jobs/378193136#L813) (there are a few instances of f-strings which won't work on 2.7!)

Reply:
Its true, we totally don't support 2.7 😬, everything on travis is run through a `conda` environment.

I typically let it run on both interpreters in travis in case someone bootstraps the project with their local Python, before using an env. But at the moment, no environment-less invocation of `python` exists, so it's a meaningless run.